lib/ansible/modules/windows/win_template.py

```diff
@@ -13,6 +13,7 @@
 ---
 module: win_template
 version_added: "1.9.2"
+short_description: Create file from template or Backup given file.
```

Comment:
Should just be what was on the original `short_description` in the template_common file.